api/docs/api.yml

```diff
@@ -0,0 +1,122 @@
+swagger: '2.0'
+info:
+  description: >
+    Scoreboard API powers the Scoreboard. It provides the OSM users data to the Scoreboard and allow grouping and searching of users and campaigns. It also provides insight into OSM top stats.
+  version: '0.1.0'
+  title: Scoreboard API
+consumes:
+  - application/json
+produces:
+  - application/json
+tags:
+  - name: users
+    description: Users
+  - name: campaigns
+    description: Campaigns
+  - name: stats
+    description: Top Stats
+paths:
+  /scoreboard/api/users:
+    get:
+      summary: list of OSM users
+      tags:
+        - users
+      parameters:
+      - in: query
+        name: page
+        type: integer
+        description: The pagination parameter (each page is limit to 25 results)
+        required: false
+      - in: query
+        name: search
+        type: string
+        description: Is used for searching the users display name
+        required: false
+      - in: query
+        name: country
+        type: string
+        description: "list of countries to return. To search for more than on country, use comma. Example: canada,france"
```

Comment:
This should be "List of countries to filter on, using a 2 letter country code. Example 'CA,US' for Canada, United States"